src/components/movie-view/similar-movies.jsx

```diff
@@ -0,0 +1,24 @@
+import { MovieCard } from "../movie-card/movie-card";
+import { Row, Col } from "react-bootstrap";
+
+export const SimilarMovies = ({movies, movie}) => {
+    let similarMovies = movies.filter((m) => movie.genre === m.genre && m !== movie)
```

Comment:
```suggestion
    let similarMovies = movies.filter((m) => movie.genre === m.genre && m.id !== movie.id)
```
Can you try this by pasting this `({} ==={})` in your browser console.

Reply:
Done! True, this is definitely not the way to compare objects.😅